int32 numPars
float32[] belief
